man/journal-upload.conf.xml

```diff
@@ -129,6 +129,25 @@
         <listitem><para>Takes a boolean value, enforces using compression without content encoding negotiation.
         Defaults to <literal>false</literal>.</para>
 
+        <xi:include href="version-info.xml" xpointer="v258"/></listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>Header=</varname></term>
+
+        <listitem><para>HTTP header which should be added to each request to URL. Headers are expected to
+        be unquoted and it's expected to be no more than one header per each option.</para>
```

Comment:
hmm Set-Cookie is often used multiple times within the same request. any particular reason why to be stricter here? why not just allow that?

Reply:
"unquoted", what do you mean by that?

you should mention that C-style escapes are supported.